packages/web3-core/test/unit/web3_extend.test.ts

```diff
@@ -0,0 +1,78 @@
+﻿/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { Address, BlockTag } from 'web3-types';
+import { Web3Context } from './reexported_web3_context';
+
+declare module './reexported_web3_context' {
+	interface Web3Context {
+		L2Module: {
+			getL2Balance(address: Address, blockTag: BlockTag): Promise<bigint>;
+		};
+
+		getL0Balance(address: Address, blockTag: BlockTag): Promise<bigint>;
+	}
+}
```

Comment:
I test this locally and it works the same:
```suggestion
import { Web3Context } from '../../src/web3_context';

declare module '../../src/web3_context' {
	interface Web3Context {
		L2Module: {
			getL2Balance(address: Address, blockTag: BlockTag): Promise<bigint>;
		};

		getL0Balance(address: Address, blockTag: BlockTag): Promise<bigint>;
	}
}
```

Reply:
I added this file if some one want to see it as one of the example in this way.